lib/resolver.coffee

```diff
@@ -41,6 +42,9 @@ yeomanResolver = require('yeoman-environment/lib/resolver')
 exports.getNodeModulesPaths = ->
 	paths = yeomanResolver.getNpmPaths()
 
+	if os.platform() isnt 'win32'
+		paths.unshift('/usr/local/lib/node_modules')
```

Comment:
No, I mean `require('module').globalPaths`, check https://github.com/nodejs/node-v0.x-archive/blob/061151c/lib/module.js#L526 and https://github.com/nodejs/node/blob/master/lib/module.js#L488 - it's been around a while :P  Presumably it's undocumented because it's pretty internal workings that not many people would need